cosmosdb/sql/ps-sql-ru-update.ps1

```diff
@@ -1,10 +1,14 @@
 # Update RU for an Azure Cosmos DB SQL (Core) API database or container
```

Comment:
Unable to get this script to run. Errored out with:

```
Set-AzResource : {"code":"Unauthorized","message":"Unable to authenticate the request. The request requires valid user authentication.\r\nActivityId: 13b39691-2d72-4ef8-927e-585f7a4974cc, Microsoft.Azure.Documents.Common/2.5.1"}
At C:\Users\sttramer\src\azure-docs-powershell-samples\cosmosdb\sql\ps-sql-ru-update.ps1:20 char:1
+ Set-AzResource -ResourceType $databaseThroughputResourceType `
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Set-AzResource], ErrorResponseMessageException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementat
```

Reply:
Unclear why you received this error. I've retested the script and it is working ok. Made some small changes to this script but otherwise is ok.